detd/ipc.proto

```diff
@@ -12,6 +12,7 @@ message StreamQosRequest {
 	uint32 txmax = 8;
 	bool   setup_socket = 9;
 	uint32 basetime = 10;
+	string qdiscmap = 11;
```

Comment:
We will need to remove qdiscmap and pass the specific options instead.

It is also possible that the application does not specify the options, and then the framework will pick it up.